### ISSUE TITLE
abnormalFunctionality: inheres in vs. inheres in part

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/upheno#361]
I know this has been explained to death, but I still am not sure which is correct.  And searching through HP, it is almost evenly split between the 2.
Ex.
'has part' some 
    (functionality
     and ('inheres in' some hypothalamus)
     and ('has modifier' some abnormal))
vs.
'has part' some 
    (functionality
     and ('inheres in part of' some spleen)
     and ('has modifier' some abnormal))

And the decreasedFunctionalityInLocation and increasedFunctionalityInLocation pattern suggestions don't.  Sorry, I hate to be a pain, but I don't know which it should be. Thanks. 
@srobb1 @matentzn @cmungall @balhoff @nicolevasilevsky

**Comment by matentzn** [comment]
Good catch. This is the main topic of thursdays Phenotype call. Let’s postpone discussions till then.

**Comment by nicolevasilevsky** [comment]
pattern_name: abnormalFunctionalityOfAnatomicalEntity
pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalFunctionalityOfAnatomicalEntity.yaml
description: "Dysfunction of an entity, such as an anatomical entity, like an organ, for example HP_0030964 'Abnormal aortic physiology'"

classes:
  functionality: PATO:0001509
  abnormal: PATO:0000460
  anatomical entity: UBERON:0001062

relations: 
  inheres_in: RO:0000052
  has_modifier: RO:0002573
  has_part: BFO:0000051

annotationProperties:
  exact_synonym: oio:hasExactSynonym 

vars:
  anatomical_entity: "'anatomical entity'"

name:
  text: "abnormality of %s physiology"
  vars:
   - anatomical_entity

def:
  text: "Any functional anomaly of the %s."
  vars:
    - anatomical_entity

equivalentTo:
  text: "'has_part' some ('functionality' and ('inheres_in' some %s) and ('has_modifier' some 'abnormal'))"
  vars:
    - anatomical_entity

@obophenotype/phenotype-editors could you all review this pattern above and sign off on it in the [spreadsheet](https://docs.google.com/spreadsheets/d/1KRx7aWd53PaPEmY-xDAurcNsfHP2EFTbY-J6LzP_dvk/edit#gid=442249367).
Nico hopes to have these all reviewed by mid-March, in time for the workshop at Biocuration2019.

**Comment by matentzn** [comment]
@nicolevasilevsky Would you mind creating the `abnormalFunctionalityOfPartOfAnatomicalEntity.yaml` pattern as well? just the same as `abnormalFunctionalityOfAnatomicalEntity.yaml`, just using `inheres in part of`.

**Comment by nicolevasilevsky** [comment]
@matentzn I created a PR: https://github.com/obophenotype/upheno/pull/568
